src/components/forms/GSScriptField.jsx

```diff
@@ -56,9 +56,23 @@ export default class GSScriptField extends GSFormField {
     this.handleCloseDialog();
   };
 
+  // handleAdminTextPreview = () => {
+  //   "I am the AdminTextPreview"
+  //   // const value = this.state.sampleContact;
+  //   // this.props.onChange(value);
+    
+  // }
+
   renderDialog() {
+    // console.log("I'M FROM GSScriptField Props")
+    // console.log(this.props)
+    // // console.log("I'M FROM GSScriptField State")
+    // // console.log(this.state)
+    // console.log("I'M FROM GSScriptField THIS")
+    // console.log(this)
```

Comment:
This was the initial spot that I was digging through to see where the AdminTextPreview method could go.  I eventually decided that this was outside the scope of this file.